frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -30,7 +30,8 @@ def serialize(obj):
         elif hasattr(obj, '__dict__'):
             return serialize(obj.__dict__)
         else:
-            logger.warning('unable to serialize object: {}'.format(obj))
+            if obj:
```

Comment:
> if isinstance(obj, (bool, six.integer_types, float, six.binary_type, six.text_type)):
            return obj

You are right about the boolean part. did not see that. thanks!
> Another question, will you not fix this bug?

Fair enough. I will fix it myself! Hopefully tonight(IST).
I have to verify if the `serialize` method is really needed.